modules/sr/robot/motor_devices.py

```diff
@@ -37,12 +39,10 @@ def set_speed(self, speed):
 
 class Gripper(MotorBase):
 
-    def __init__(self, webot: Robot, motor_name: str) -> None:
+    def __init__(self, webot: Robot, motor_names: Tuple[str, str]) -> None:
```

Comment:
I wonder if instead, this should just be 2 separate arguments?

Reply:
Eh. They do en up in a list and are treated as just elements of that list though, so it feels more fitting to accept them like this.